app/views/posts/index.html.erb

```diff
@@ -12,6 +12,14 @@
         <span><em><b>Comments: <%= post.comments_counter%></b></em></span>
         <span><em><b>Likes: <%= post.likes_counter%></b></em></span>
       </div>
+      <ul>
+        <% if can? :destroy, Post %>
+          <li><%= button_to "del", user_post_path(post.author, post.id), method: :delete  %></li>
+        <%end%>
```

Comment:
- [ ] Let's fix the relationships for a post table, comment table, and like table. We should instruct a parent table (post table) that when a post with a like(s) or with a comment(s) is deleted, even its likes or comments should be deleted. We can do this at the model level or in our migration files. I got the error in the image below.

![like](https://user-images.githubusercontent.com/53124548/224068163-476e4473-8f56-437c-95b2-955fa496a23e.png)

Reply:
Okay thank you for the feedback @elisha2kyakpo1, we will address them all.